ui_framework/components/info_panel/_info_panel.scss

```diff
@@ -1,8 +1,59 @@
 .kuiInfoPanel {
-  padding: 20px;
+  padding: 14px 20px 18px;
   line-height: $lineHeight;
+  border: 2px solid;
+}
+
+/**
+ * 1. TODO: Pick a hex value instead of making these colors translucent.
```

Comment:
Yeah, based on what you're saying here it sounds like it's actually not a TODO. I'm not very particular on this though.